src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -213,7 +213,7 @@
     <Compile Include="System\Diagnostics\ProcessThread.Windows.cs" />
     <Compile Include="System\Diagnostics\ProcessThreadTimes.cs" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true'">
+  <ItemGroup Condition=" '$(TargetsUnixOrBrowser)' == 'true'">
```

Comment:
@akoeplinger do we have data if any of the entry points work

Reply:
According to the test results: `Tests run: 241, Errors: 0, Failures: 30, Skipped: 118. Time: 3.016593s`

Here's a list of tests that are passing. Looks like mostly tests that verify exception behavior, I don't think it makes sense to try to make this library work and we should throw PNSE instead.

```
System.Diagnostics.Tests.ProcessCollectionTests.TestModuleCollectionBehavior
System.Diagnostics.Tests.ProcessModuleTests.ModuleCollectionSubClass_DefaultConstructor_Success
System.Diagnostics.Tests.ProcessModuleTests.Modules_GetMultipleTimes_ReturnsSameInstance
System.Diagnostics.Tests.ProcessModuleTests.Modules_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessStartInfoTests.Domain_GetSetUnix_ThrowsPlatformNotSupportedException
System.Diagnostics.Tests.ProcessStartInfoTests.FileName_Set_GetReturnsExpected(fileName: \&quot;\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.FileName_Set_GetReturnsExpected(fileName: \&quot;filename\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.FileName_Set_GetReturnsExpected(fileName: null)
System.Diagnostics.Tests.ProcessStartInfoTests.InitializeWithArgumentList
System.Diagnostics.Tests.ProcessStartInfoTests.LoadUserProfile_GetSetUnix_ThrowsPlatformNotSupportedException
System.Diagnostics.Tests.ProcessStartInfoTests.Password_GetSetUnix_ThrowsPlatformNotSupportedException
System.Diagnostics.Tests.ProcessStartInfoTests.PasswordInClearText_GetSetUnix_ThrowsPlatformNotSupportedException
System.Diagnostics.Tests.ProcessStartInfoTests.TestArgumentsProperty
System.Diagnostics.Tests.ProcessStartInfoTests.TestEnvironmentProperty
System.Diagnostics.Tests.ProcessStartInfoTests.TestEnvironmentVariables_Environment_DataRoundTrips
System.Diagnostics.Tests.ProcessStartInfoTests.TestEnvironmentVariablesPropertyUnix
System.Diagnostics.Tests.ProcessStartInfoTests.TestUseShellExecuteProperty_SetAndGet
System.Diagnostics.Tests.ProcessStartInfoTests.UnintializedArgumentList
System.Diagnostics.Tests.ProcessStartInfoTests.UserName_Set_GetReturnsExpected(userName: \&quot;\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.UserName_Set_GetReturnsExpected(userName: \&quot;domain\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.UserName_Set_GetReturnsExpected(userName: null)
System.Diagnostics.Tests.ProcessStartInfoTests.Verb_Set_GetReturnsExpected(verb: \&quot;\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.Verb_Set_GetReturnsExpected(verb: \&quot;verb\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.Verb_Set_GetReturnsExpected(verb: null)
System.Diagnostics.Tests.ProcessStartInfoTests.WindowStyle_Set_GetReturnsExpected(style: Hidden)
System.Diagnostics.Tests.ProcessStartInfoTests.WindowStyle_Set_GetReturnsExpected(style: Maximized)
System.Diagnostics.Tests.ProcessStartInfoTests.WindowStyle_Set_GetReturnsExpected(style: Minimized)
System.Diagnostics.Tests.ProcessStartInfoTests.WindowStyle_Set_GetReturnsExpected(style: Normal)
System.Diagnostics.Tests.ProcessStartInfoTests.WindowStyle_SetNoSuchWindowStyle_ThrowsInvalidEnumArgumentException(style: -1)
System.Diagnostics.Tests.ProcessStartInfoTests.WindowStyle_SetNoSuchWindowStyle_ThrowsInvalidEnumArgumentException(style: 4)
System.Diagnostics.Tests.ProcessStartInfoTests.WorkingDirectory_Set_GetReturnsExpected(workingDirectory: \&quot;\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.WorkingDirectory_Set_GetReturnsExpected(workingDirectory: \&quot;workingdirectory\&quot;)
System.Diagnostics.Tests.ProcessStartInfoTests.WorkingDirectory_Set_GetReturnsExpected(workingDirectory: null)
System.Diagnostics.Tests.ProcessTests.CloseMainWindow_NotStarted_ReturnsFalse_UWPNonWindows
System.Diagnostics.Tests.ProcessTests.ExitTime_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.GetProcesses_EmptyMachineName_ThrowsArgumentException
System.Diagnostics.Tests.ProcessTests.GetProcesses_InvalidMachineName_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.GetProcesses_RemoteMachinePath_ReturnsExpected
System.Diagnostics.Tests.ProcessTests.GetProcessesByName_NoSuchProcess_ReturnsEmpty
System.Diagnostics.Tests.ProcessTests.GetProcesseses_NullMachineName_ThrowsArgumentNullException
System.Diagnostics.Tests.ProcessTests.HandleCount_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.HasExited_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Kill_EntireProcessTree_True_CalledOnCallingProcess_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Kill_EntireProcessTree_True_ProcessNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Kill_NotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.LongProcessNamesAreSupported
System.Diagnostics.Tests.ProcessTests.MachineName_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.MaxWorkingSet_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.MinWorkingSet_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.NonpagedSystemMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.NonpagedSystemMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PagedMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PagedMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PagedSystemMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PagedSystemMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PeakPagedMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PeakPagedMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PeakVirtualMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PeakVirtualMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PeakWorkingSet_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PeakWorkingSet64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PriorityClass_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PrivateMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PrivateMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.PriviledgedProcessorTime_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.ProcessName_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.SafeHandle_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.SessionId_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.StandardInput_GetNotRedirected_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Start_Disposed_ThrowsObjectDisposedException
System.Diagnostics.Tests.ProcessTests.Start_EmptyFileName_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Start_HasStandardErrorEncodingNonRedirected_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Start_HasStandardInputEncodingNonRedirected_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Start_HasStandardOutputEncodingNonRedirected_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.Start_NullStartInfo_ThrowsArgumentNullExceptionException
System.Diagnostics.Tests.ProcessTests.StartInfo_GetOnRunningProcess_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.StartInfo_SetNull_ThrowsArgumentNullException
System.Diagnostics.Tests.ProcessTests.StartTime_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.TestInvalidPriorityClass(priorityClass: 0)
System.Diagnostics.Tests.ProcessTests.TestInvalidPriorityClass(priorityClass: 96)
System.Diagnostics.Tests.ProcessTests.TestModules
System.Diagnostics.Tests.ProcessTests.TestStartWithBadWorkingDirectory
System.Diagnostics.Tests.ProcessTests.TestWindowApisUnix
System.Diagnostics.Tests.ProcessTests.TotalProcessorTime_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.UserProcessorTime_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.VirtualMemorySize_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.VirtualMemorySize64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.WorkingSet_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessTests.WorkingSet64_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessThreadTests.Threads_GetNotStarted_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessWaitingTests.WaitForExit_NotDirected_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessWaitingTests.WaitForExitAsync_NotDirected_ThrowsInvalidOperationException
System.Diagnostics.Tests.ProcessWaitingTests.WaitForInputIdle_NotDirected_ThrowsInvalidOperationException
```

Reply:
Yeah, there is not much except Process in the assembly. We could turn the assembly into PNSE one